tool2schema/type_schema.py

```diff
@@ -96,9 +101,9 @@ def to_json(self) -> dict:
         Return the json schema for this type.
         """
         fields = {
-            "type": self._get_type(),
             "items": self._get_items(),
             "enum": self._get_enum(),
+            **self._get_type(),
```

Comment:
This is now inconsistent with the other two methods. I was thinking of having them return dictionaries as well, but then overriding methods would have to repeat the `"enum"`/`"items"` keyword. Not sure which way is better

Reply:
This is probably fine for now.